### DISCUSSION TITLE
Localised Image Assets

### DISCUSSION THREAD

**Discussion by teejayhh** [SwiftGen/SwiftGen#995] [Ideas]
I am just wondering if it is possible to implement localised assets as its a feature of asset catalogues. There are ways of getting around it but it would be awesome if it would come inbuilt. I might look into this and see if I can change the generator template and put a solution forward.

**Comment by djbe** [comment]
From what I could gather from the Apple documentation, there are no changes to the API for creating an image/color/… from an asset catalog. For example `UIImage(named:)` will itself check if the asset is localized, and if so, use the runtime language to load the right file.
